js/palette.js

```diff
@@ -443,15 +445,25 @@ function Palettes(canvas, refreshCanvas, stage, cellSize, refreshCanvas, trashca
             circles = showButtonHighlight(
                 palettes.buttons[name].x + r, palettes.buttons[name].y + r, r,
                 event, palettes.scale, palettes.stage);
+            
+            /*add tooltip for palette buttons*/
+            console.log("r: " + r);
+            palette_text = new createjs.Text(name, "20px Arial", "black");    
```

Comment:
Sorry. We use a function _ for translation, so that, for example, _('rhythm') would get translated into 'ritmo' in Spanish. In your case, we want to translate the palette name, so we'd use _(name)

See https://github.com/walterbender/musicblocks/blob/master/js/utils.js#L196 for more details.